src/propagators.jl

```diff
@@ -83,6 +83,13 @@ end
 
 QuKrylov() = QuKrylov(@compat Dict(:NC=>10))
 
+immutable QuODESolvers <: QuPropagatorMethod
+    method
```

Comment:
In sense do we aim at every solver having a type, something on the lines of 
```julia
immutable QuODE45 <: QuPropagatorMethod
options::Dict
end
```
 Any clue on how we could generate different types would also be helpful.